api/v1alpha1/BUILD.bazel

```diff
@@ -0,0 +1,85 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "cluster_defaults.go",
+        "cluster_types.go",
+        "condition_types.go",
+        "groupversion_info.go",
+        "pod_container_types.go",
+        "volume.go",
+        "volume_types.go",
+        "zz_generated.deepcopy.go",
+    ],
+    importpath = "github.com/cockroachdb/cockroach-operator/api/v1alpha1",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@com_github_pkg_errors//:go_default_library",
+        "@io_k8s_api//apps/v1:go_default_library",
+        "@io_k8s_api//core/v1:go_default_library",
+        "@io_k8s_apimachinery//pkg/apis/meta/v1:go_default_library",
+        "@io_k8s_apimachinery//pkg/runtime:go_default_library",
+        "@io_k8s_apimachinery//pkg/runtime/schema:go_default_library",
+        "@io_k8s_sigs_controller_runtime//pkg/scheme:go_default_library",
+    ],
+)
+
+ETCD = "//hack/bin:etcd"
+
+go_test(
+    name = "go_default_test",
+    srcs = [
+        "cluster_defaults_test.go",
+        "cluster_types_test.go",
+        "suite_test.go",
+        "volume_test.go",
+    ],
+    # TODO use these values in the unit test instead of the hardcoded values
```

Comment:
Is there an easy way to inject these values into `$PATH`?

Reply:
So we have options.  

I am using https://github.com/cockroachdb/cockroach-operator/blob/5928ca90d499d672ac54fa68dcbecb7e3748849e/pkg/testutil/paths/paths.go#L40 to modify the `PATH`.  But I have the names of the binaries hardcoded in the unit test here:
https://github.com/cockroachdb/cockroach-operator/blob/5928ca90d499d672ac54fa68dcbecb7e3748849e/api/v1alpha1/suite_test.go#L63

My thought was to pass in the names of the binaries as parameters. 
The reason I have the test adjust the `PATH` is that only the `PATH` for this test suite is modified..

The other option is changing the PATH for bazel.  Which is messy, to say the least.